bx_django_utils/feature_flags/data_classes.py

```diff
@@ -0,0 +1,100 @@
+from collections.abc import Iterable
+from typing import Optional
+
+from django.core.cache import cache
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from bx_django_utils.feature_flags.exceptions import NotUniqueFlag
+from bx_django_utils.feature_flags.utils import validate_cache_key
+
+
+class State(models.IntegerChoices):
+    """"""  # noqa - don't add in README
+
+    ENABLED = 1, _('enabled')
+    DISABLED = 0, _('disabled')
+
+
+class FeatureFlag:
+    """
+    A feature flag that persistent the state into django cache.
```

Comment:
This is unsafe to use, right?
A) If the Redis is unavailable (which we handle in our application), then everything would go down.
B) If the cache is cleared the data is lost! It should persist to database instead.

Reply:
Yes and no: It depends on `initial_enabled` value ;) 
Think it's a feature request to store everything into database. I create https://github.com/boxine/bx_django_utils/issues/90 for this.

Reply:
@phihag I catch an error on `cache.get()` and use the initial value as fallback.